### PR TITLE
Factory Girl, Data Cleaner, orientation script spec begining, 

### PR DESCRIPTION
**Pull request by Ewiseman**
journey#show table, participant#new drop down also added.

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -8,39 +8,58 @@
...
+          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
+            <span class="sr-only">Toggle navigation</span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+          </button>
+          <a class="navbar-brand" href="/">Journey Seva</a>
+        </div>  
+        <div id="navbar" class="navbar-collapse collapse">
+          <ul class="nav navbar-nav navbar-right">
```
Comment (eliotsykes) [inline app/views/layouts/application.html.erb]: Lines 24 to 38 could be cleaned up a little. There's an empty block and duplicate `user_signed_in?` checks.

File: app/views/participants/show.html.erb [hunk 2]
```diff
@@ -24,7 +24,7 @@
   <div class="row">
     <div class="col-md-3">
       <% if policy(Welcomescript.new).create? %>
-        <% if @welcomescript.save %>
+        <% if !@welcomescript.persisted? %>
```
Comment (eliotsykes) [inline app/views/participants/show.html.erb]: For future reference, the alternative I was trying to remember yesterday that would do the same job here is:

```html
<% if @welcomescript.new_record? %>
```

Comment (Ewiseman) [inline app/views/participants/show.html.erb]: Hi Eliot, 

Thanks for letting me know.  Did you happen to get my bloc message about this line of code?

Comment (eliotsykes) [inline app/views/participants/show.html.erb]: Hi Eric, just replied on Bloc.

Comment (Ewiseman) [inline app/views/participants/show.html.erb]: Sweet! Thanks!

**Pull request merged**